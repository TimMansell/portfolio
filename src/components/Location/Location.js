import React from 'react';

import './Location.scss';

export const Location = () => {
	return <h4 className="location">Melbourne, Australia</h4>;
}

export default Location;