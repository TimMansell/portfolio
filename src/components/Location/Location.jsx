import React from 'react';

import styles from './Location.module.scss';

export const Location = () => {
  return <div className={styles.location}>Melbourne, Australia</div>;
};

export default Location;
