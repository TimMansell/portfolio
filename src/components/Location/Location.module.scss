@import 'variables';
@import 'utilities/pxtorem';

.location {
  font-size: pxtorem(16);
  opacity: 0;
  color: $secondary-text-colour;
  font-weight: 300;
  letter-spacing: pxtorem(1);
  line-height: 1.5;
  text-transform: uppercase;
  text-shadow: 1px 1px darken($primary-colour, 10%);
  animation: location 0.5s ease-in-out;
  animation-delay: $animation-location-delay;
  animation-fill-mode: forwards;

  @media (min-width: $bp-md) {
    font-size: pxtorem(18);
    text-shadow: 2px 2px $tertiary-text-colour;
  }
}

@keyframes location {
  from {
    opacity: 0;
    transform: translateY(-10px);
  }

  to {
    opacity: 1;
    transform: translateY(0);
  }
}
