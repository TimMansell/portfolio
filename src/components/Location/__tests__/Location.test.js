import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import Location from '../Location';

describe('Location', () => {
  it('should render my component', () => {
    render(<Location />);
  });

  it('should match snapshot', () => {
    const { asFragment } = render(<Location />);

    expect(asFragment()).toMatchSnapshot();
  });
});
