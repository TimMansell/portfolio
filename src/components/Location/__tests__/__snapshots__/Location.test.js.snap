// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="location"
  >
    Melbourne, Australia
  </div>
</DocumentFragment>
`;
